main.py

```diff
@@ -38,8 +43,25 @@ def create_client(region_id):
 
     client = Client(config)
     return client
-
-
+# Get all images in builds.json and check the build meta.json to see
+# if we had an aliyun artifact created
+def parse_relase(release):
+    releases = []
+    jsonurl = urlopen("%srhcos-%s/builds.json" % (REDIRECTOR_URL, release))
+    buildjson = json.loads(jsonurl.read())
+    for build in (buildjson['builds']):
+        buildid = (build['id'].replace('.','')).replace('-','')
+        # Look only for builds after the aliyun inclusion
+        # TODO: we can improve it keeping a record for the build we already checked
+        if int(buildid) >= FIRSTRELEASE[build['arches'][0]]:
+            metajsonURL = ("%srhcos-%s/%s/%s/meta.json" % (REDIRECTOR_URL, release, build['id'], build['arches'][0]))
+            jsonurl = urlopen(metajsonURL)
+            metajson = json.loads(jsonurl.read())
+            if 'aliyun' in metajson['images']:
```

Comment:
We should probably check for the presence of uploaded images to Aliyun, rather than the artifact itself, since it is possible to build the artifact but not upload to the cloud.

```suggestion
            if 'aliyun' in metajson:
```